### PR TITLE
Slack hack using selenium

### PR DESCRIPTION
**Pull request by ajayrahul97**

### CODE REVIEW

File: slackhack.ipynb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    }
+   ],
+   "source": [
+    "import selenium\n",
+    "from selenium import webdriver\n",
+    "from selenium.webdriver.support.ui import WebDriverWait\n",
+    "from selenium.webdriver.common.keys import Keys\n",
+    "\n",
+    "\n",
+    "b = webdriver.Chrome(\"/home/ajayrahul/chromedriver\")\n",
```
Comment (ProgVal) [inline slackhack.ipynb]: You should fix this

**Comment by ProgVal** [comment]
Isn't the purpose of this to flood Slack?

**Comment by ajayrahul97** [comment]
yea..just a fun hack :P

**Comment by karlingen** [comment]
Need this. Please merge.

**Pull request merged**